docs/sources/installation/mac.md

```diff
@@ -1,116 +1,306 @@
-page_title: Installation on Mac OS X
-page_description: Instructions for installing Docker on OS X using boot2docker.
+page_title: Installation on Mac OS X 
+page_description: Instructions for installing Docker on OS X using boot2docker. 
 page_keywords: Docker, Docker documentation, requirements, boot2docker, VirtualBox, SSH, Linux, OSX, OS X, Mac
 
-# Installing Docker on Mac OS X
+# Install Docker on Mac OS X
 
-> **Note:**
-> Docker is supported on Mac OS X 10.6 "Snow Leopard" or newer.
+Because the Docker daemon uses Linux-specific kernel features, you can't run
+Docker natively in OS X.  Instead, you must install the Boot2Docker application.
+Boot2Docker includes a VirtualBox VM, Docker itself, and the Boot2Docker
+management tool.
 
-Because the Docker Engine uses Linux-specific kernel features, you'll need to use a
-lightweight virtual machine (VM) to run it on OS X. You use the OS X Docker client to
-control the virtualized Docker Engine to build, run, and manage Docker containers.
+The Boot2Docker management tool is a lightweight Linux virtual machine made
+specifically to run the Docker daemon on Mac OS X. The VirtualBox VM runs
+completely from RAM, is a small ~24MB download, and boots in approximately 5s.
```

Comment:
oh dear no.

The Boot2DOcker management tool is a small application which creates a virtual machine using the Boot2Docker Linux distribution.

There are actually 2 separate products called Boot2Docker - the application, and the Linux distribution.

The Boot2Docker Linux distribution does _not_ (normally) run completely from RAM. The Boot2Docker management tool creates a persistence partition that stores your docker images and containers - this persists over reboots of both b2d and your osx/windows/linux box.

sadly, the 5s boot time is also not correct - the first boot can take up to several minutes, as it needs to initialise the persistence partition and several certificates (and saying its fast has caused many support problems, as it sets users up for thinking that there's something wrong the with their b2d when they've just installed.